### ISSUE TITLE
Add redirect to domain name

### ISSUE THREAD

**Issue by kevinrobinson** [mit-teaching-systems-lab/threeflows#359]
Because the domain is used in querying for data, you can access different data based on the domain that's used for the learner experience or the researcher experience.

We should probably start redirecting everything to https://teachermoments.teachingsystemslab.org/ to make this simpler going forward.  We could also update the query for researcher data to not care about the specific domain (just the path part of the URL).

FWIW @keving17 I ran into this just now, trying to use https://teachermoments.teachingsystemslab.org/ for data entered at the herokuapp domain. :)

**Comment by kevinrobinson** [comment]
@keving17 Yep, I think both steps would be good - add redirection on the server so that if the domian isn't `teachermoments.teachingsystemslab.org` it redirects to that domain.  That simplifies things going forward.  And updating the query for researcher data so that it doesn't look at the domain name will make that work retroactively for past data.

I don't think having separately domains was intentional, it's just historical from before we had our own domain, and people still using older links.

**Comment by keving17** [comment]
Hmmm. So I'm working on getting rid of the domain in the queries right now and I can't figure out how to work around string manipulation. From what I can see, the correct way to do the psql query for partial matches is
```
SELECT * FROM access where url LIKE '%/teachermoments/turner?WWA04062018'
```

but I can't seem to add the `%` character to the `location` variable

Also, how exactly does the redirect work? I don't see anything that seems relevant in index.js

**Comment by keving17** [comment]
@kevinrobinson any ideas here?

**Comment by kevinrobinson** [comment]
@keving17 For querying, this may be helpful to check out: https://stackoverflow.com/questions/19471756/how-to-make-a-like-search-in-postgresql-and-node-js.

For the redirect, grep for `https` in https://github.com/mit-teaching-systems-lab/threeflows/blob/master/server/index.js and see how it's redirecting from http to https.  We could do something similar where if the domain doesn't match `ENV['CANONICAL_DOMAIN']` (eg, `teachermoments.teachingsystemslab.org`) then we redirect to that domain plus the request's path (eg, `teachermoments.teachingsystemslab.org${request.path}` or something like that).
